packages/gestalt-design-tokens/tokens/classic/sema-color-light.json

```diff
@@ -16,6 +16,12 @@
       "error": {
         "value": "{color.red.pushpin.500.value}"
       },
+      "hover": {
```

Comment:
we can't touch this file. classic files that arent' web component tokens cannot change

Reply:
fix this and I'll r+

Reply:
Done!